connectivity/suite.go

```diff
@@ -727,13 +732,25 @@ func Run(ctx context.Context, ct *check.ConnectivityTest, addExtraTests func(*ch
 		)
 
 	ct.NewTest("egress-gateway").
-		WithCiliumEgressGatewayPolicy(egressGatewayPolicyYAML).
+		WithCiliumEgressGatewayPolicy(egressGatewayPolicyYAML, check.CiliumEgressGatewayPolicyParams{}).
+		WithIPRoutesFromOutsideToPodCIDRs().
```

Comment:
Last nit (no need to re-run the CI): could you protect the test case with `if ct.Params().IncludeUnsafeTests {
` (as other `WithIPRoutesFromOutsideToPodCIDRs()` tests do)?